apis/bases/memcached.openstack.org_memcacheds.yaml

```diff
@@ -134,6 +134,9 @@ spec:
                 items:
                   type: string
                 type: array
+              tlsSupport:
+                description: Whether TLS is supported by the memcached instance
+                type: boolean
```

Comment:
maybe an enum to indicate the verify mode?

Reply:
as discussed on slack, if we introduce client certs, we could change it. its a status and atm not consumed direct. instead its consumed by keystone using the func added in this PR